src/Microsoft.Azure.SignalR.Management/Obsolete/ServiceManager.cs

```diff
@@ -55,7 +56,7 @@ public async Task<IServiceHubContext> CreateHubContextAsync(string hubName, ILog
 
         public void Dispose()
         {
-            ((IDisposable)_serviceProvider).Dispose();
+            (_serviceProvider as IDisposable).Dispose();
```

Comment:
why change it? 
((IDisposable)_serviceProvider)?.Dispose();

Reply:
Allow `ServiceProvider` to be a customized implementation and not disposable.

Reply:
then you need a `?`